build/README.md

```diff
@@ -24,12 +24,15 @@ To test this plugin in applications that use it before publishing a release, use
 
 To update the project readme, edit the README.md in ./build and run `build:doc`. This runs Typedoc and appends the generated type information to the readme file, which is then published to the ./docs folder.
 
-## Publishing new versions
+## Commiting and publishing new versions
```

Comment:
Curious as to why we need two versions of the README?

Reply:
Great question! The readme is autogenerated via `npm build:doc` by combining the readme in `./build` with generated docs. So a change in the `build` readme always has a commensurate change in `docs`.

It'd be nice to autogenerate documentation on a git action, for example when PRs are merged, to a) save devs seeing it in PRs and b) ensure the autogenerated documentation is never stale.